### PR TITLE
Added Google Maps API with markers

### PR DESCRIPTION
**Pull request by shivanighanta**
Features:

Google Maps API
- Markers for important locations on campus
- When user clicks the marker, a picture pops up in a window with the description of the location
- Title of the location appears when hovering over the marker

Link to Live Website: https://sghanta-step-2020-com2.appspot.com/

image

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -57,92 +61,104 @@ <h2>Aspiring Software Engineer</h2>
...
+                  <p class="struct">
+                    Long Drives<br />
+                    Avocados<br />
+                    Lavendar<br />
+                    Sweater Weather<br />
+                  </p>
+                  </div>
+                </center>
+                <div class="image">
+                  <center><img class="img-responsive" src="images/favorites.png" height="250" width="250" /></center>
```
Comment (ginellegaisano) [inline portfolio/src/main/webapp/index.html:115]: For img-response, you could set the height/weight in CSS, to share between this img and above

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -1,20 +1,24 @@
 <!DOCTYPE html>
 <html>
```
Comment (jbir789) [inline portfolio/src/main/webapp/index.html:2]: Nit: add a lang such as: ```<html lang="en-US">```

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -57,92 +61,104 @@ <h2>Aspiring Software Engineer</h2>
...
-          </div>
-          </center>
-          <div class="image">
-            <center><img class="img-responsive" src="images/courses.png" height="250" width="250"/></center>
+                Data Structures and Algorithms (CS 225)<br />
+                Computer Architecture (CS 233)<br />
+                Discrete Structures (CS 173)<br />
+                Numerical Methods (CS 357)<br />
+                Intro to Computer Science (CS 125) <br />
+                </div>
```
Comment (jbir789) [inline portfolio/src/main/webapp/index.html:69]: Your div should be moved below `</center>` to match the opening tags

File: portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -57,92 +61,104 @@ <h2>Aspiring Software Engineer</h2>
             <center>
```
Comment (jbir789) [inline portfolio/src/main/webapp/index.html:61]: center is deprecated, so prefer using CSS to center the text

File: portfolio/src/main/webapp/index.html [hunk 5]
```diff
@@ -57,92 +61,104 @@ <h2>Aspiring Software Engineer</h2>
...
+            <div class="about">
+              <center>
+                <h2>Hobbies</h2>
+                <p class="struct">
+                  Making DIY Projects<br />
+                  Reading Self-Improvement Books<br />
+                  Oil Painting<br />
+                  Sipping Herbal Tea<br />
+                </p>
+                </div>
```
Comment (jbir789) [inline portfolio/src/main/webapp/index.html:91]: Same here, you need to move it below `</center>`

File: portfolio/src/main/webapp/index.html [hunk 6]
```diff
@@ -57,92 +61,104 @@ <h2>Aspiring Software Engineer</h2>
...
-          </div>
-          </center>
-          <div class="image">
-            <center><img class="img-responsive" src="images/hobbies.png" height="250" width="250"/></center>
+      <!-- Hobbies -->
+      <div class="info-row info-row1">
+        <div class="container">
+          <div class="row">
+            <div class="about">
+              <center>
```
Comment (jbir789) [inline portfolio/src/main/webapp/index.html:83]: Supposedly, center isn't permitted inside a div. If you remove it, does it actually change anything?

File: portfolio/src/main/webapp/index.html [hunk 7]
```diff
@@ -57,92 +61,104 @@ <h2>Aspiring Software Engineer</h2>
...
-              <!-- TODO: Change to unordered list & keep consistent formatting -->
               <p class="struct">
-                Long Drives<br>
-                Avocados<br>
-                Lavendar<br>
-                Sweater Weather<br>
+                <body onload="getRandomQuote()">
+                  <div id="quote-container"></div>
+                  </p>
+                  </div>
```
Comment (jbir789) [inline portfolio/src/main/webapp/index.html:129]: Sam here

File: portfolio/src/main/webapp/index.html [hunk 8]
```diff
@@ -57,92 +61,104 @@ <h2>Aspiring Software Engineer</h2>
...
+                  </center>
+                  <!-- Comment Box-->
+                  <div class="container_comment">
+                    <form method="POST" action="/data">
```
Comment (jbir789) [inline portfolio/src/main/webapp/index.html:133]: I think you need to use lower case `post`

File: portfolio/src/main/webapp/index.html [hunk 9]
```diff
@@ -57,92 +61,104 @@ <h2>Aspiring Software Engineer</h2>
...
+                          <option value="USA">USA</option>
+                          <option value="Canada">Canada</option>
+                          <option value="Australia">Australia</option>
+                        </select>
+                        <label for="subject">Subject</label>
+                        <textarea
+                          id="subject"
+                          name="subject"
+                          placeholder="Write something.."
+                          style="height: 200px;"
```
Comment (jbir789) [inline portfolio/src/main/webapp/index.html:150]: inline style isn't allowed per the lint check, please prefer using the CSS file instead.

File: portfolio/src/main/webapp/index.html [hunk 10]
```diff
@@ -57,92 +61,104 @@ <h2>Aspiring Software Engineer</h2>
...
+                        ></textarea>
+                        <input type="submit" value="Submit" />
```
Comment (jbir789) [inline portfolio/src/main/webapp/index.html:152]: Prefer using `<button type="submit">Submit</ button>` instead

File: portfolio/src/main/webapp/index.html [hunk 11]
```diff
@@ -57,92 +61,104 @@ <h2>Aspiring Software Engineer</h2>
...
+                    </form>
+                  </div>
+                </body>
               </p>
-          </div>
-          </center>
-          <div class="image">
-            <center><img class="img-responsive" src="images/favorites.png" height="250" width="250"/></center>
+            </center>
           </div>
```
Comment (jbir789) [inline portfolio/src/main/webapp/index.html:159]: The following tags apparently don't have a matching opening tag. Are they needed?

**Comment by jbir789** [comment]
Really good PR description!

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (shivanighanta) [reply portfolio/src/main/webapp/index.html]: Setting the height and width for the images in CSS made it off center for some reason.

File: portfolio/src/main/webapp/index.html [hunk 12]
```diff
@@ -98,51 +97,45 @@ <h2>Hobbies</h2>
...
       </div>
     </div>
+    <!-- Map -->
+    <div id="map" class="map_format"></div>
     <!-- Generate Random Quote -->
     <div class="info-row info-row1">
-      <center>
-        <p class="struct">
-          <body onload="getRandomQuote()">
+      <body onload="getRandomQuote()">
```
Comment (jbir789) [inline portfolio/src/main/webapp/index.html:118]: You can't have a `<body>` inside a `<div>`, could you remove the enclosing div?

Comment (shivanighanta) [reply portfolio/src/main/webapp/index.html:118]: I wanted a specific format for the quotes so when I remove the <div> the formatting goes away.

Comment (jbir789) [reply portfolio/src/main/webapp/index.html:118]: Could you set the class on the body?

Comment (shivanighanta) [reply portfolio/src/main/webapp/index.html]: ok sure

**Pull request merged**